drake/examples/simple_continuous_time_system.cc

```diff
@@ -0,0 +1,81 @@
+
+/// Simple Continuous Time System Example
+///
+/// This is meant to be a sort of "hello world" example for the
+/// drake::system classes.  It defines a very simple continous time system,
+/// simulates it from a given initial condition, and plots the result.
+
+#include "drake/systems/analysis/simulator.h"
+#include "drake/systems/framework/leaf_system.h"
+
+namespace drake {
+namespace examples {
+
+/// Simple Continuous Time System
+/// \begin{gather*}
+///     \dot{x} = -x + x^3 \\
+///     y = x
+/// \end{gather*}
+class SimpleContinuousTimeSystem : public systems::LeafSystem<double> {
+ public:
+  SimpleContinuousTimeSystem() {
+    this->DeclareOutputPort(systems::kVectorValued,
+                            1,  // dimension of output (y) = 1
+                            systems::kContinuousSampling);
+  };
+  ~SimpleContinuousTimeSystem() override{};
+
+  // xdot = -x + x^3
+  void EvalTimeDerivatives(
+      const systems::Context<double>& context,
+      systems::ContinuousState<double>* derivatives) const override {
+    double x = context.get_continuous_state()->get_state().GetAtIndex(0);
```

Comment:
would love to have one less level of redirection in here.  (e.g. context.get_continuous_state_vector().GetAtIndex(0)).  

also, perhaps rename ContinousState<T>.get_state to ContinuousState<T>.get_vector?